packages/db-objects-redis/lib/objects/objectsInRedisClient.js

```diff
@@ -2927,7 +3001,7 @@ class ObjectsInRedisClient {
 
                 let objs;
                 try {
-                    objs = await this.client.evalsha([this.scripts.filter, 5, this.objNamespace, params.startkey, params.endkey, m[1], cursor]);
+                    objs = await this.client.evalsha([this.scripts.filter, 6, this.objNamespace, params.startkey, params.endkey, m[1], cursor, `${this.setNamespace}object.type.${m[1]}`]);
```

Comment:
fehlt nicht hier auch check for useSets?? (und nächste Fälle ach)

Reply:
Man kann da extra Logik bauen, aber letztendlich ist das alles backward compatible, da nur am Ende ein Parameter mehr übergeben wird, der wird halt in den alten Luas nicht angezogen.. würde es so lassen ums nicht noch komplizierter zu machen...